transport/x/grpc/util_test.go

```diff
@@ -66,3 +67,21 @@ func TestProcedureNameFunctions(t *testing.T) {
 		})
 	}
 }
+
+func TestProcedureNameEmpty(t *testing.T) {
+	_, _, err := procedureNameToServiceNameMethodName("")
+	require.Error(t, err)
+	_, err = procedureNameToFullMethod("")
+	require.Error(t, err)
+}
+
+func TestToFullMethodEmptyServiceName(t *testing.T) {
+	require.Equal(t, "/__default__/foo", toFullMethod("", "foo"))
```

Comment:
(Ideally the gRPC method would be equivalent to YARPC procedure, and not force Protobuf opinions about procedure name structure on other encodings. But, I’m sure we can’t have nice things.)